packages/terra-time-input/src/TimeInput.module.scss

```
@@ -53,16 +53,16 @@
     &.is-focused {
       @include focus-styles;
       background-color: var(--terra-time-input-focus-background-color, #fff);
-      background-size: var(--terra-time-input-focus-keyboard-animation-size, 0 0);
+      background-size: var(--terra-time-input-focus-keyboard-background-size, 0 0);
```

Let's drop the word `keyboard`. This [matches changes we did to terra-form-input variables](https://github.com/cerner/terra-core/blob/master/packages/terra-form-input/docs/UPGRADEGUIDE.md.

```suggestion
      background-size: var(--terra-time-input-focus-background-size, 0 0);
```